qutebrowser/browser/qutescheme.py

```diff
@@ -435,6 +435,20 @@ def qute_settings(url):
     return 'text/html', html
 
 
+@add_handler('bindings')
+def qute_bindings(url):
+    """Handler for qute://bindings View qute bindings."""
+    assert url
```

Comment:
I assume you're doing this because the argument is unused? Rename it to `_url` instead.